derrida/books/views.py

```diff
@@ -239,6 +239,53 @@ def get_queryset(self):
                 solr_facet = self.form.solr_field(facet)
                 # filter the query: facet matches any of the terms
                 sqs = sqs.filter(**{'%s__in' % solr_facet: search_opts[facet]})
+        # request range facets for References, adapated from logic
+        # above for Instances
+        # get max/min from database to specify range start & end values
+        # TODO: should probably cache max/min values so we don't have
```

Comment:
Any thoughts on caching these?

Reply:
I think that's caching is probably  the way to go. I wanted a working system that mirrored the one for `Instance` before refactoring anything too heavily beyond what had to be, so I'll turn to that!